src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingSubmissionAndResultIntegrationTest.java

```diff
@@ -134,7 +135,7 @@ public void tearDown() {
     /**
      * The student commits, the code change is pushed to the VCS.
      * The VCS notifies Artemis about a new submission.
-     *
+     * <p>
```

Comment:
Are all these `<p>` changes intended?

Reply:
I removed the p's. I'm not sure how the got there